### ISSUE TITLE
HTTP endpoint subscription error

### ISSUE THREAD

**Issue by rodrigocarlospires** [p4tin/goaws#205]
Subscription to **http** endpoint from config file (goaws.yaml) don't work. So a **sns topic** configuration like that:

```
...
Topics:
    - Name: my_topic
      Subscriptions:
        - Protocol: http
          EndPoint: http://backend/my_listener
          TopicArn: arn:aws:sns:us-east-1:000000000000:my_topic
          FilterPolicy: '{"event": ["my_event"]}'
```
doesn't work!
From config file just subscription to SQS queues (*QueueName*) works! (on goaws container boot).

**TopicArn** like ListTopics response:
```
TopicArn: arn:aws:sns::000000000000:my_topic
```
also doesn't work :(

This way I need to subscribe manually (aws cli) to http endpoints every time the goaws is restarted.

Some solution to that?

Thanks!

**Comment by p4tin** [comment]
Fixed in version 0.2.1 - https://github.com/p4tin/goaws/releases/tag/v0.2.1 -or- 
```
docker pull docker.io/pafortin/goaws:latest
```

**Comment by rodrigocarlospires** [comment]
Thanks @p4tin for fix this \o/.
More one thing if possible: the FilterPolicy didn't work for http subscription, is this sintax ok?:

```
FilterPolicy: '{"event": ["my_event"]}'
```
